### ISSUE TITLE
Horizontal scroll bar appears on TOC even when there's lots of room

### ISSUE THREAD

**Issue by DragaDoncila** [napari/napari-sphinx-theme#80]
Even with a wide window the scroll bar on the left-side TOC appears even when the text is short and there's lots of space (I think it might be the search box?):

![image](https://user-images.githubusercontent.com/17995243/156457285-ff072f8d-5c69-456f-b5d7-e3e095abba76.png)

It would be nice if we could just have it appearing for very long names or not at all (wrap the names)

**Comment by tlambert03** [comment]
same here

**Comment by liaprins-czi** [comment]
Yeah, ideally there wouldn't be a side scrollbar at all ever, and long names should just wrap

**Comment by psobolewskiPhD** [comment]
This is still a thing, just more subtle because the scroll bar is at the bottom of the page, after #134 
![image](https://github.com/napari/napari-sphinx-theme/assets/76622105/e9ea18b5-d231-44f8-9a03-d3fc52487202)
It doesn't happen on the pydata-sphinx-theme site, so it must be somehow related to our setup.

**Comment by melissawm** [comment]
I can't reproduce on Firefox/Linux, so that may be browser-related. Will take a look

**Comment by psobolewskiPhD** [comment]
Yup! I don't see it in Chrome, just Safari.

**Comment by psobolewskiPhD** [comment]
Interestingly, it's not on all pages.
Contributing and API reference are fine. But homepage, usage, plugins, and community it's there.
Likewise pydata-sphinx read the docs site it's fine.

**Comment by psobolewskiPhD** [comment]
Gah, this isn't actually fixed by pydata 15.3
It's just that the spurious scroll bar disappears when there is stuff to scroll vertically -- makes no sense. But see:
![image](https://github.com/napari/napari-sphinx-theme/assets/76622105/db2fafaa-a01c-4966-871f-928083bbe1c3)
vs.
![image](https://github.com/napari/napari-sphinx-theme/assets/76622105/e6620ed2-b2f1-40d3-9809-64544de12e31)

**Comment by psobolewskiPhD** [comment]
It's not a priority :)

**Comment by melissawm** [comment]
Not reproducible anymore, please reopen if that's not the case.
